apps/voting/contracts/Voting.sol

```diff
@@ -268,10 +366,21 @@ contract Voting is IForwarder, AragonApp {
     /**
     * @dev Return the state of a voter for a given vote by its ID
     * @param _voteId Vote identifier
+    * @param _voter Address of the voter
     * @return VoterState of the requested voter for a certain vote
     */
     function getVoterState(uint256 _voteId, address _voter) public view voteExists(_voteId) returns (VoterState) {
-        return votes[_voteId].voters[_voter];
+        return _voterState(votes[_voteId], _voter);
+    }
+
+    /**
+    * @dev Return the issuer of a given vote by its ID
+    * @param _voteId Vote identifier
+    * @param _voter Address of the voter
+    * @return Address of the issuer of the voter's vote
+    */
+    function getVoteIssuer(uint256 _voteId, address _voter) public view voteExists(_voteId) returns (address) {
+        return _voteIssuer(votes[_voteId], _voter);
     }
```

Comment:
Super nitpick: can we re-order the new internal and private functions so they're mostly in alphabetical order?

Unless something should be logically grouped, I find alphabetical definitions a pretty easy way to recognize where to put new functions :).

Reply:
Any reason to do that only for the `internal` and `private` functions? I suggest tackling convention-related changes in all apps at once, otherwise we will never have the consistency we're actually trying to get. WDYT?

Reply:
Hmm yes, I didn't notice some of the new public functions weren't alphabetized.

I agree, we can do it all later (and most of the apps _should_ be fairly standard now...). I guess the biggest offenders are `setRepresentative()`, `withinOverruleWindow()`, and `isRepresentativeOf()`, which could be moved around a bit.